spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/onnx/ConstantSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.onnx
+
+import com.intel.analytics.bigdl.tensor.Tensor
+import org.scalatest.{FlatSpec, Matchers}
+import com.intel.analytics.bigdl.utils.serializer.ModuleSerializationTest
+
+
+class ConstantSpec extends FlatSpec with Matchers {
+
+  "Constant" should "work" in {
+    val value = Tensor[Float](512, 3, 224, 224).rand()
+    val constant = Constant(value)
+    val out = constant.forward()
+    out should be(value)
+  }
+
+}
+
+
+class AbsSerialTest extends ModuleSerializationTest {
```

Comment:
why AbsSerialTest in this file ?

Reply:
Thanks.
Wrong copy & paste, will fix it.